src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -225,6 +228,27 @@ private ConfigurationSetting<AlgorithmName> GetHashAlgorithmName(IConfiguration
         };
     }
 
+    private ConfigurationSetting<ComplianceStandardType> GetComplianceStandard(IConfiguration configuration)
+    {
+        // Convert to ComplianceStandard enum value.
+        var oldValue = configuration.ComplianceStandard;
+        var newValue = ComplianceStandardType.FromString(oldValue?.Value?.ToString());
+
+        // Compliance standard is only supported for ManifestInfo value of SPDX 3.0 and above.
+        if (!newValue.Equals(ComplianceStandardType.None) && !configuration.ManifestInfo.Value.Any(mi => mi.Equals(Constants.SPDX30ManifestInfo)))
+        {
+            throw new ValidationArgException($"Please use SPDX >=3.0 or remove the ComplianceStandard parameter.");
```

Comment:
This level of messaging will get hard to maintain going forward. Would it be enough to give them a message like this? $"Compliance standard {newValue} is not supported with manifest format {configuration.ManifestInfo.Value}. Please use a supported combination."

Reply:
Yes I think that should be enough info.